app/assets/stylesheets/forms.scss

```diff
@@ -0,0 +1,118 @@
+// Form Elements
+// -------------
+// Inputs
+// Textarea
+// Checkboxes
+// Buttons
+
+fieldset {
+  padding: 0;
+  border: none;
+  margin-bottom: 24px;
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  ul {
+    list-style-type: none;
+    margin: 0;
+    padding-left: 0;
+  }
+}
+
+legend {
+  color: #202428; // smoke-90
+  font-weight: 500;
+}
+
+label,
+input,
+textarea {
+  box-sizing: border-box;
+  margin: 0;
+  width: 100%;
+}
+
+// Input Fields
+label {
+  color: #202428; // smoke-90
+  display: flex;
+  flex-direction: column;
+  font-weight: 500; // medium
+  margin-bottom: 16px;
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+}
+
+input[type='text'],
+textarea {
+  background-color: #f9fafa;
```

Comment:
If YES, lets put them only on top of each file instead of `/stylesheets/variables.scss` which is more global.

Reply:
@ludwiczakpawel Should I leave this as it is since we're waiting for the Crayons variables to get merged in later?

Reply:
should be fine, sure.

Reply:
I ended up removing the file `forms.scss` completely since we already have the Crayons version. Instead, I brought a lot of the form styles into the `onboarding-modal.scss` file so that we reduce duplicate styles and files.